app/models/language.rb

```diff
@@ -1,6 +1,7 @@
 class Language < ApplicationRecord
   belongs_to :casa_org
-  has_and_belongs_to_many :users
+  has_many :user_languages
+  has_many :users, through: :user_languages
```

Comment:
scope this to casa org?

Reply:
Will do. Still getting familiar with the data models and relationships.

Reply:
How do you mean? Each language already belongs to a `casa_org` with a distinct record in the database. The uniqueness validation only cares about not associating the same `Language` record to the same `User` record more than once. On the _edit profile_ page the only languages available to select are those belonging to same organization as the user. So they can't select a language outside the scope of their organization.

Reply:
that makes sense, works for me :)